src/app/component/auth/sign-in-new/sign-in-new.component.html

```diff
@@ -32,12 +68,30 @@ <h2>{{'user.auth.sign-in.fill-form' | translate}}</h2>
                alt="show or hide password">
         </span>
       </div>
+      <div class="validation-password-error"
+           *ngIf="passwordControl.invalid && (passwordControl.dirty||passwordControl.touched)">
+        <div *ngIf="passwordControl.errors.required">
+          {{'user.auth.sign-in.password-is-required' | translate}}
+        </div>
+        <div *ngIf="passwordControl.errors.minlength">
+          {{'user.auth.sign-in.password-must-be-at-least-8-characters-long' | translate}}
+        </div>
+      </div>
+      <div class="validation-password-error"
+           *ngIf="passwordErrorMessageBackEnd!=null">
+        {{passwordErrorMessageBackEnd}}
+      </div>
+      <div class="alert-general-error"
+           *ngIf="backEndError!=null">
+        {{backEndError}}
+      </div>
```

Comment:
On line 119 please add appropriate functionality to open 'Sign up' form

Reply:
Fixed